### ISSUE TITLE
The script crashes after clicking on the " countdown" button

### ISSUE THREAD

**Issue by kevinfredrick** [SmartBotBlack/farmers-world-bot#3]
Hi, 
I have a saw and two phasing rods. if one of these reaches 00:00:00, the script crashes and needs to copy and paste again to get it to work.
I think it is because of the waiting time after clicking on the Mining button, not sure.
could you please take a quick look and fix this issue. 

Many thanks.

**Comment by anter-faree** [comment]
That's right. I think it is because the waiting time after clicking on the mining button takes a long time ( up to one minute in some cases).

I hope the developer will take a look and modify the code soon.

**Comment by voodee** [comment]
@kevinfredrick @ANter-xidian 
They released an update today. Check if the script works.
